test/http-delivery-mocha.js

```diff
@@ -0,0 +1,71 @@
+// http-delivery-mocha.js
+
+var chai = require('chai');
+var chaiAsPromised = require('chai-as-promised');
+chai.should();
+chai.use(chaiAsPromised);
+
+var http = require('http');
+var _ = require('underscore');
+var noflo = require('noflo');
+var test = require('./common-test');
+var httpDelivery = require('../components/http-delivery');
+
+describe('http-delivery', function() {
+    it("should receive a request using a sub-graph", function() {
+        return new Promise(function(done, fail) {
+            test.createNetwork({
+                delivery: "rdf-components/http-delivery-server"
+            }).then(function(network){
+                afterEach(_.once(network.stop.bind(network)));
+                network.graph.addInitial("Hello World!", 'delivery', 'content');
+                network.graph.addInitial('text/plain', 'delivery', 'type');
+                network.graph.addInitial(1337, 'delivery', 'listen');
```

Comment:
I assume 1337 is the port number.  Might be good to define that as a var above, as a means of documenting it.